hack/istio/multicluster/env.sh

```diff
@@ -77,6 +77,12 @@ NETWORK2_ID="network-west"
 # Deploy a single kiali or a kiali per cluster
 SINGLE_KIALI="${SINGLE_KIALI:-true}"
 
+# Deploy just in one cluster
+SINGLE_CLUSTER="${SINGLE_CLUSTER:-false}"
```

Comment:
It is different, to prevent the installation the permissions in the cluster 2. You might install 1 single Kiali, but if is multi cluster it will require permissions in cluster 2, and the cluster won't exist for single cluster.

Reply:
But, you mean, just in the ENV script? Because there is also modifications in deploy_kiali. If single kiali is true it will fail because of this permissions in the cluster 2, that the context does not exist and fails.